src/brewtarget.cpp

```diff
@@ -388,10 +388,16 @@ const QDir Brewtarget::getConfigDir(bool *success)
    return dir.absolutePath() + "/";
 
 #elif defined(Q_OS_WIN) // Windows OS.
+   if (!qApp)
+   {  //QApplication instance doesn't exist yet
+      if (success)
+         *success = false;
+      return QString();
```

Comment:
This should never happen if the code is correct, so add an error log message here.